examples/nips17_adversarial_competition/run_attacks_and_defenses.sh

```diff
@@ -16,7 +16,11 @@ MAX_EPSILON=16
 # Prepare working directory and copy all necessary files.
 # In particular copy attacks defenses and dataset, so originals won't
 # be overwritten.
-WORKING_DIR=$(mktemp -d)
+if [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
Please replace $OSTYPE with ${OSTYPE} so the entire file will follow the same style.